doc/modules/cross_validation.rst

```diff
@@ -223,10 +222,11 @@ learned using :math:`k - 1` folds, and the fold left out is used for test.
 Example of 2-fold cross-validation on a dataset with 4 samples::
 
   >>> import numpy as np
-  >>> from sklearn.cross_validation import KFold
+  >>> from sklearn.model_selection import KFold
 
-  >>> kf = KFold(4, n_folds=2)
-  >>> for train, test in kf:
+  >>> X = np.ones(4)
```

Comment:
No no that is a valid point! keeping `X = ["a", "b", "c", "d"]` is a better choice! That being said don't you feel users should also know that `split` just takes in data so it could get the length from it and hence as long as the length is same, the data itself doesn't matter... we need to somehow connote that in our example too...(? Maybe this is overthinking it :P but anyway `X = ["a", "b", "c", "d"]` is indeed a better choice)